qiita_ware/commands.py

```diff
@@ -60,46 +58,35 @@ def submit_EBI(preprocessed_data_id, action, send, fastq_dir_fp=None):
                         info={'ebi_submission': preprocessed_data_id})
         raise
 
-    # other steps
-    output_dir = fastq_dir_fp + '_submission'
-    sample_template = ebi_submission.sample_template
-    prep_template = ebi_submission.prep_template
-    preprocessed_data = ebi_submission.preprocessed_data
-
-    samp_fp = join(fastq_dir_fp, 'sample_metadata.txt')
-    prep_fp = join(fastq_dir_fp, 'prep_metadata.txt')
-
-    sample_template.to_file(samp_fp, demux_samples)
-    prep_template.to_file(prep_fp, demux_samples)
-
-    # Get specific output directory and set filepaths
-    get_output_fp = partial(join, output_dir)
-    study_fp = get_output_fp('study.xml')
-    sample_fp = get_output_fp('sample.xml')
-    experiment_fp = get_output_fp('experiment.xml')
-    run_fp = get_output_fp('run.xml')
-    submission_fp = get_output_fp('submission.xml')
-
-    if not isdir(output_dir):
```

Comment:
Yes, before we had different folders for different things and now we have 1 single folder.